de.haeckerfelix.gradio.json

```diff
@@ -0,0 +1,135 @@
+{
+    "app-id": "de.haeckerfelix.gradio",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "gradio",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Tracker1",
+        "--socket=pulseaudio",
+        "--talk-name=org.gtk.vfs", "--talk-name=org.gtk.vfs.*",
+        "--talk-name=org.gnome.ScreenSaver",
+        "--talk-name=org.gnome.SettingsDaemon",
+        "--talk-name=org.mpris.MediaPlayer2",
```

Comment:
Why would it need to talk to this? It owning its own name below should be all for mpris.